master_navigator/include/master_navigator/MasterNavigator.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
```

Comment:
We should decide on a naming convention for private member variables to help my small brain. I've always used the leading underscore but I've seen most ROS stuff uses the trailing underscore. I don't really care which convention we use but I find it a lot easier to read

Reply:
We need to create a naming convention for things overall. We can chat tmr

Reply:
I'll leave this open until we determine what we are going to do.